### ISSUE TITLE
Is it working on daily experimental build?

### ISSUE THREAD

**Issue by kosirm** [overmindstudios/BlenderUpdaterCLI#15]
I just downloaded latest release ([BlenderUpdaterCLI 1.5](https://github.com/overmindstudios/BlenderUpdaterCLI/releases/download/v1.5/BlenderUpdaterCLI.exe)) and tried to make it work following your [video on youtube](https://youtu.be/HJWh0kSRBGg)... but nothing happens, bat file opens and then self-close immediately... with no result.
I use Windows 10 -  64bit and I would like to update [daily experimental blender build](https://builder.blender.org/download/).

**Comment by kosirm** [comment]
I also tryed to run it with python from console and I get this error:
```
$ python BlenderUpdaterCLI.py -p C:\Programs\Blender-Daily\blender -b 2.83 -o windows -a x64
Traceback (most recent call last):
  File "BlenderUpdaterCLI.py", line 21, in <module>
    from progress.bar import IncrementalBar        
ModuleNotFoundError: No module named 'progress'
```

**Comment by tobkum** [comment]
can you try running your .bat file that runs the BlenderUpdaterCLI.exe via the command line and see if there's any output?
The error message in your second post says that you don't have the progress python module installed, but that is only necessary if you're running the .py file directly - not when running the .exe

EDIT: also, make sure that you have write permissions in your `C:\Programs` folder

**Comment by kosirm** [comment]
Thanks! Running from command line works like a charm! It also created config.ini file, so I think that now it will also run with a click on a bat file. Problem solved! Thanks again.

**Comment by tobkum** [comment]
That's weird though... I'll reopen and investigate when I have time - but good that it works for you now!
Can you post your .bat file?

**Comment by tobkum** [comment]
My best guess is that it should work when its something along those lines:
`%~dp0\BlenderUpdaterCLI.exe -p C:\Programs\Blender-Daily\blender -b 2.83 -o windows -a x64`

Explanation: the `%~dp0` makes sure the batch file is running in the correct directory, no matter where it's called from, given that it's placed in the same folder as the `BlenderUpdaterCLI.exe`

**Comment by kosirm** [comment]
Thanks for additional hint! I will change my bat file accordingly. 
My bat file until now was this:
`BlenderUpdaterCLI.exe -p blender -b 2.83 -o windows -n`

I removed -a flag, because it is giving me unrecognized flag error, I also found this 1.4 release note:
- removed architecture flag (only 32bit supported from now on)

... and really it is working perfectly now, even from bat file! It is also asking me on the end:
`This version is already installed. Continue anyways? [Y]es or [N]o:`

Thanks you so much for this huge time-saver!!!
